dolphin/drivers/dell_emc/vmax/client.py

```diff
@@ -165,6 +165,7 @@ def list_volumes(self, storage_id):
                     "status": status,
                     "original_id": vol['volumeId'],
                     "wwn": vol['wwn'],
+                    "provisioning_policy": constants.ProvisioningPolicy.THIN,
```

Comment:
We can assume THIN for all vmax3 or above models , but before that there is a support of THICK volumes.. so how do we derive the logic ?  one way could be to check whether dataDev=true !!

Reply:
As discussed, currently no known way to identify type for vmax.